react/react1/week3/todo-app/src/contexts/ToDosContext.js

```diff
@@ -0,0 +1,73 @@
+import { createContext, useEffect, useReducer } from 'react';
+
+export const ToDosContext = createContext();
+
+const initialState = {
+  todos: [],
+};
+
+const toDosReducer = (state, action) => {
+  switch (action.type) {
+    case 'GET_TODOS':
+      return { todos: action.payload };
```

Comment:
It can be nice to have the actions either here up top or in a separate file, so you can manage them centrally and an onlooker can also get an easy overview with what is going on instantly. 
Just something like:
``` 
actions = {GET_TODOS : "GET_TODOS", ...}
```

Reply:
I created a new folder called **utils**, and created **an actions.js** file that will store all the **action types**.